DOCUMENTATION.md

```diff
@@ -4196,7 +4196,7 @@ The name must be a string; the optional doc argument can have any type.
     "start_timestamp": str,
     "end_timestamp": str,
     "push_timestamp": str,
-    "image_size": int,
+    "image_size": str,
```

Comment:
Just to make sure as I'm unfamiliar with python:
This is actually a big int, is `str` the correct type for that?